server/route.go

```diff
@@ -992,13 +992,47 @@ func (c *client) processRemoteSub(argo []byte, hasOrigin bool) (err error) {
 	// Lookup the account
 	// FIXME(dlc) - This may start having lots of contention?
 	accountName := string(args[0+off])
-	acc, _ := srv.LookupAccount(accountName)
+	// Lookup account while avoiding fetch.
+	// A slow fetch delays subsequent remote messages. It also avoids the expired check (see below).
+	// With all but memory resolver lookup can be delayed or fail.
+	// It is also possible that the account can't be resolved yet.
+	// This does not apply to the memory resolver.
+	// When used, perform the fetch.
+	staticResolver := true
+	if res := srv.AccountResolver(); res != nil {
+		if _, ok := res.(*MemAccResolver); ok {
+			staticResolver = true
+		} else {
+			staticResolver = false
+		}
+	}
+	acc := (*Account)(nil)
```

Comment:
Although valid, a simple `var acc *Account` reads better I think, and more inline with rest of the code.

Reply:
changed